#  Integrated value of the velocity of the accelerometer

uint64 timestamp                # time since system start (microseconds)

uint64 timestamp_sample         # the timestamp of the raw data (microseconds)

float32[3] integrated_value_velocity                    # the integral of the accelerometer values
